src/tokens/formats/json-formats.js

```diff
@@ -0,0 +1,70 @@
+const jsonVariables = {
+  name: 'rvt/json/variables',
+  formatter: function(dictionary) {
+    const allProperties = dictionary.allProperties;
+    const properties = dictionary.properties;
+    // eslint-disable-next-line prefer-const
+    let categories = Object.keys(properties);
+
+    // Wrapper is the string that will be built into the final output
+    let wrapper = ``;
+    wrapper += `{\n`;
+
+    // Output holds the output from each category loop
+    let output = ``;
+    categories.forEach(function(item) {
```

Comment:
See comment above. Could be a little clearer to do the following:

```suggestion
    Object.keys(properties).forEach(item => {
```

You could alternatives eliminate the `properties` variable declaration too if you want:

```js
Object.keys(dictionary.properties).forEach(item => {
  // Do stuff with each property/dictionary entry
});
```